pkg/cli/admin/release/info.go

```diff
@@ -1563,6 +1567,35 @@ func describeBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string, format s
 			bugIDs.Insert(commit.Bugs...)
 		}
 	}
+	return bugIDs, utilerrors.NewAggregate(errs)
+}
+
+func listBugs(out, errOut io.Writer, diff *ReleaseDiff, dir string) error {
+	if diff.To.Digest == diff.From.Digest {
+		return fmt.Errorf("releases are identical")
+	}
+	bugs, err := getBugSet(out, errOut, diff, dir)
+	if err != nil {
+		fmt.Fprintf(errOut, "error: %v\n", err)
+		return kcmdutil.ErrExit
+	}
+	for id := range bugs {
+		fmt.Fprintln(out, id)
```

Comment:
This means on the consumer's end we will need to read a file line-by-line and parse integers, which seems less straightforward than if this tool output JSON or something. WDYT?